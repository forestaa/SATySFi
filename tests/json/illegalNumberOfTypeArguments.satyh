type 'a 'b x = 'a 'b option
