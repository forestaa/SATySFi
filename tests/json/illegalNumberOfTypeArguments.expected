{
  "file": "illegalnumberoftypearguments.satyh",
  "range": {
    "start": { "line": 1, "character": 15 },
    "end": { "line": 1, "character": 27 }
  },
  "message":
    "! [Type Error] 'option' is expected to have
1 type argument(s),\n    but it has 2 type argument(s) here."
}
